geomesa-features/geomesa-feature-avro/src/main/scala/org/locationtech/geomesa/features/avro/AvroSimpleFeatureUtils.scala

```diff
@@ -29,6 +29,7 @@ object AvroSimpleFeatureUtils {
 
   val FEATURE_ID_AVRO_FIELD_NAME: String = "__fid__"
   val AVRO_SIMPLE_FEATURE_VERSION: String = "__version__"
+  val AVRO_SIMPLE_FEATURE_USERDATA: String = "__userdata__"
 
   // Increment whenever encoding changes and handle in reader and writer
   val VERSION: Int = 2
```

Comment:
There may be some merit. We could keep them the same though we'd have to bump them in tandem with each other... @elahrvivaz any opinions? Doesn't bother me too much but if its more logical and easy to follow i can start with v2